workshops/base_views.py

```diff
@@ -206,3 +206,19 @@ def get_success_url(self):
             return next_url
         else:
             return default_url
+
+
+class PrepopulationSupportMixin:
+
+    def get_initial(self):
+        return self.request.GET.dict()
```

Comment:
I'm :+1: for better control by the programmer. We can have `prepopulated_fields` (or similar) class variable that lists all fields we allow for prepopulation.